package.json

```diff
@@ -49,6 +50,7 @@
     "react-test-renderer": "15.6.1",
     "regenerator-runtime": "0.10.5",
     "screener-storybook": "0.7.10",
+    "slimerjs": "0.10.3",
```

Comment:
I know this is not the point but just curious, why slimerjs over phantom? You will need something like `xvfb` on CI

It may also be fun to play with headless chrome https://medium.com/@dschnr/using-headless-chrome-as-an-automated-screenshot-tool-4b07dffba79a